examples/rllib/utils.py

```diff
@@ -73,10 +73,9 @@ def step(self, action_dict):
         for index, agent_id in enumerate(self._ordered_agent_ids)
     }
     done = {'__all__': timestep.last()}
-    info = {}
 
-    observations = utils.timestep_to_observations(timestep)
-    return observations, rewards, done, info
+    observations, info = utils.timestep_to_observations(timestep)
+    return observations, rewards, done, done, info
```

Comment:
Does the second done need to not have __all__? https://docs.ray.io/en/latest/_modules/ray/rllib/env/multi_agent_env.html#make_multi_agent

Reply:
I don't know but this works for me.